data/original/payment-methods.csv

```diff
@@ -33,8 +33,10 @@ online,kcp_banktransfer,Bank Transfer via KCP,65,41,100,60,256,256,world,
 online,kcp_creditcard,Credit Card via KCP,65,41,100,60,256,256,world,
 online,kcp_payco,PayCo,65,41,100,60,256,256,world,
 online,klarna,Klarna,65,41,100,60,256,256,world,
+online,mobilepay,MobilePay,65,41,100,60,256,256,world,
 online,molpay_points,MOLPoints via MOLPay,65,41,100,60,256,256,world,
 online,multibanco,Multibanco,65,41,100,60,256,256,world,
+online,onlineBanking_PL,Przelewy24,65,41,100,60,256,256,world,

```

Comment:
Why this case?
Also, why this name and not `przelewy24`?

Reply:
it is defined as onlineBanking_PL in Adyen: 
https://docs.adyen.com/payment-methods/online-banking-poland/api-only/#make-payment

But I think we can change it to przelewy24 and use onlineBanking_PL internally when making a request to Adyen
What do you think?

Reply:
I agree - this is our reference, not Adyen's.

Did you check what other PSPs are saying?

Reply:
Renamed to przelewy24